src/hooks/transactions/useGetTokenDetails.tsx

```diff
@@ -87,12 +98,16 @@ export function useGetTokenDetails({
     : '';
 
   return {
```

Comment:
use a type here

Reply:
The type is being used, it's called `TokenOptionType`. 👍